src/libs/actions/Policy.ts

```diff
@@ -3237,6 +3238,94 @@ function openPolicyDistanceRatesPage(policyID?: string) {
     API.read(READ_COMMANDS.OPEN_POLICY_DISTANCE_RATES_PAGE, params);
 }
 
+function createPolicyDistanceRate(customUnitRate: Rate, customUnitID: string, policyID?: string) {
```

Comment:
Let's reorder this params to be consistent with the other methods:

```suggestion
function createPolicyDistanceRate(policyID, customUnitID: string, customUnitRate: Rate) {
```